week-1-ts-ecosystem/w01e01-typescript-basics/src/user/addressType.ts

```diff
@@ -0,0 +1,4 @@
+export type addressType = {
+  street: string,
+  no: number,
+}
```

Comment:
Tutaj fajnie że address jako type ma osobny plik. Jednak wystarczy aby nazywał się:
`Address`

z dużej litery


Ja do modelu danych tego typu (biznesowego) czyli że to odpowiada za model obiektów w naszym projekcie,
osobiście używam `interface` zamiast `type`